docs/metrics/how-metrics-work.md

```diff
@@ -0,0 +1,26 @@
+---
+title: How Metrics Work on Statsig
+sidebar_label: How Metrics Work
+slug: /metrics/how-metrics-work
+---
+
+# How Metrics Work on Statsig
+
+A metric in Statsig is a numeric value for each user on a given day. This value can be aggregated across the entire user base or a subset, such as the test group of an experiment. 
+
+For example, say a user made two purchases on September 1st. This value can be aggregated across multiple users to calculate the total number of purchases across all users on September 1st.
+
+
+## Two Sources of Statsig Metrics 
+
+There are two sources of metrics in Statsig:
+1. **Raw Events** - you can convert these into metrics via [Custom Metrics](/metrics/create); Statsig also [auto-generates certain metrics](/metrics/metrics-from-events) from these events
```

Comment:
raw events themselves generate metrics without creating custom metrics e.g. logging a click will generate event_count and event_dau metrics